stats/meter32.go

```diff
@@ -67,6 +71,27 @@ func (m *Meter32) ValueUint32(val uint32) {
 	m.Unlock()
 }
 
+func (m *Meter32) ValuesUint32(val, num uint32) {
+	bin := val
```

Comment:
What's the point of having these two variables which both seem to keep the same value during the entire function?